### PR TITLE
Add IEEEST pss

### LINKED ISSUE #28: add "IEEEST" PSS model

### PR DESCRIPTION
**Pull request by rodrigomha**
To do:
- [x] Add PSS/E csv validation

This PR also adds some removal of algebraic variables when entire blocks are bypassed in the IEEEST model.

closes #28

### CODE REVIEW

File: src/models/saturation_models.jl [hunk 1]
```diff
@@ -55,6 +55,27 @@ function rectifier_function(I::T) where {T <: Real}
     end
 end
 
+function output_pss_limiter(
+    V_ss::X,
+    V_ct::Y,
+    V_cl::Float64,
+    V_cu::Float64,
+) where {X, Y <: Real}
```
Comment (jd-lara) [inline src/models/saturation_models.jl:63]:
```suggestion
) where {X  <: Real, Y <: Real}
```

File: src/models/generator_models/pss_models.jl [hunk 2]
```diff
@@ -6,6 +6,99 @@ function mass_matrix_pss_entries!(
...
+    mass_matrix[global_index[:x_p7], global_index[:x_p7]] = PSY.get_T6(pss)
+end
+
+#####################################################
+############ Obtain PSS Input Signals  ##############
+#####################################################
+
+function get_pss_input_signal(
+    ::Type{Base.RefValue{1}},
+    device_states,
```
Comment (jd-lara) [inline src/models/generator_models/pss_models.jl:27]: At least add a TODO here to set the types of the function inputs

File: src/models/generator_models/pss_models.jl [hunk 3]
```diff
@@ -6,6 +6,99 @@ function mass_matrix_pss_entries!(
...
+    pss = PSY.get_pss(dynamic_device)
+    #Obtain external states inputs for component
+    external_ix = get_input_port_ix(dynamic_device, typeof(pss))
+    ω = device_states[external_ix[1]]
+    return ω - 1.0
+end
+
+function get_pss_input_signal(
+    ::Type{Base.RefValue{2}},
+    device_states,
```
Comment (jd-lara) [inline src/models/generator_models/pss_models.jl:42]: same here

File: src/models/generator_models/pss_models.jl [hunk 4]
```diff
@@ -6,6 +6,99 @@ function mass_matrix_pss_entries!(
...
+    inner_vars,
+    ω_sys,
+    dynamic_device::DynamicWrapper{PSY.DynamicGenerator{M, S, A, TG, P}},
+) where {M <: PSY.Machine, S <: PSY.Shaft, A <: PSY.AVR, TG <: PSY.TurbineGov, P <: PSY.PSS}
+    @debug("Frequency Input for PSS not properly supported yet")
```
Comment (jd-lara) [inline src/models/generator_models/pss_models.jl:47]:
```suggestion
    @warn("Frequency Input for PSS not properly supported yet")
```

File: src/models/generator_models/pss_models.jl [hunk 5]
```diff
@@ -20,6 +113,97 @@ function mdl_pss_ode!(
...
+) where {M <: PSY.Machine, S <: PSY.Shaft, A <: PSY.AVR, TG <: PSY.TurbineGov}
+
+    #Get Signal Input Integer
+    pss = PSY.get_pss(dynamic_device)
+    input_code = PSY.get_input_code(pss)
+    #Remote bus control not supported
+
+    #Get Input Signal
+    u = get_pss_input_signal(
+        Base.RefValue{input_code},
```
Comment (jd-lara) [inline src/models/generator_models/pss_models.jl:131]:
```suggestion
        Base.RefValue{PSY.get_input_code(pss)},
```

File: src/models/generator_models/pss_models.jl [hunk 6]
```diff
@@ -20,6 +113,97 @@ function mdl_pss_ode!(
...
+    #Compute and update output signal
+    V_ss = clamp(y_out, Ls_min, Ls_max)
+    #Compute compensated terminal voltage
+    V_R = inner_vars[VR_gen_var]
+    V_I = inner_vars[VI_gen_var]
+    #To do: Figure out how to compensate terminal voltage
+    V_ct = sqrt(V_R^2 + V_I^2)
+
+    #Compute PSS output signal and update inner vars
+    #inner_vars[V_pss_var] = output_pss_limiter(V_ss, V_ct, V_cl, V_cu)
```
Comment (jd-lara) [inline src/models/generator_models/pss_models.jl:202]: Why is this portion disabled?

Comment (rodrigomha) [reply src/models/generator_models/pss_models.jl:202]: Commented that out to check some changes. My bad

**Review by jd-lara — DISMISSED** [review]

File: src/models/generator_models/pss_models.jl (hunk 2 above)
Comment (rodrigomha) [reply src/models/generator_models/pss_models.jl:27]: This is a larger issue than here. All mdl functions don't have dispatch. We need to fix that. I'll open an issue for that.

**Pull request merged**